esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -1117,25 +955,14 @@ EsdlBindingImpl::EsdlBindingImpl(IPropertyTree* cfg, const char *binding,  const
 
     try
     {
-
-        ensureSDSPath(ESDL_DEFS_ROOT_PATH);
-        ensureSDSPath(ESDL_BINDINGS_ROOT_PATH);
-
         m_esdlBndCfg.set(fetchESDLBinding(process, binding, m_esdlStateFilesLocation));
 
         if (!m_esdlBndCfg.get())
             DBGLOG("ESDL Binding: Could not fetch ESDL binding %s for ESP Process %s", binding, process);
 
-        //Subscribe to Dali anyway
         DBGLOG("ESDL Binding %s is subscribing to all /ESDL/Bindings/Binding dali changes", binding);
```

Comment:
maybe this log entry should be in the centralstore?

Reply:
Removed this log and enhanced corresponding log in central store.